docs/templates/template_reference.md

```diff
@@ -409,6 +409,19 @@ template:
     arg_name: audit
 ```
 
+#### kernel_build_config
+This template checks the configuration used to build the kernel by checking the `/boot/config-*` files.
+The template only generates a check and no remediation.
+
+- Parameters:
+
+    - **config** - The kernel configuration to check
+    - **value** - The value the configuration should have
+      When **value** is `"n"`, the check will pass when the config is absent or
+      commented out in the `/boot/config-*` files.
```

Comment:
I prefer having a explicit attribute for this absence condition. For example:

`state: absent` similar as it's done in Ansible. Otherwise it can be very confusing when only looking at the rule.yml

Reply:
I'm afraid that having two tightly coupled parameters (`value` and `state`) will be confusing.
Hmm, basically when `value` is `n` the `state` has to be `absent`, and vice-versa. 

https://unix.stackexchange.com/questions/305540/should-settings-missing-in-a-kernels-config-be-interpreted-as-setting-n-or

Reply:
I lean towards keeping just `value`, and the check does whatever it needs to do.